### PR TITLE
i18n:Feature/post editor #77

### LINKED ISSUE #77: i18n: Post Editor
As a translator I want to be able to localise these pages:
- Post Editor - https://wedance.vip/posts/-/edit

Follow i18n style guide in #70

### PR DESCRIPTION
**Pull request by larrycoal**
Closes #77

### CODE REVIEW

File: use/posts.js [hunk 1]
```diff
@@ -147,8 +148,24 @@ export const usePosts = () => {
...
+    },
+    {
+      name: 'hideComments',
+      label: t('posts.hideComments.label'),
+      admin: true,
+    },
+    {
+      name: 'hideReactions',
+      label: t('posts.hideReactions.label'),
+      admin: true,
```
Comment (razbakov) [inline use/posts.js:168]: That should be removed.

Comment (razbakov) [inline use/posts.js:168]: That should be removed.

File: use/posts.js [hunk 2]
```diff
@@ -110,32 +110,33 @@ export const usePosts = () => {
...
       component: 'TInputSelect',
+      label: t('posts.type.label'),
+      type: 'select',
       options: postTypeList,
     },
     {
       name: 'title',
       hideLabel: true,
-      placeholder: 'Title',
+      placeholder: t('posts.feildTitle.placeholder'),
```
Comment (razbakov) [inline use/posts.js:126]: You already had it fixed. Why is it again in plural?

```suggestion
      placeholder: t('post.feildTitle.placeholder'),
```

**Review by razbakov — CHANGES_REQUESTED** [review]

**Comment by razbakov** [comment]
Your active PR is https://github.com/we-dance/platform/pull/135
Please continue work there. One pull request is enough.

**Review by best-lucky1030 — COMMENTED** [review]
Looks good 👍

**Pull request closed without merging**